superset_wfs_dialect/base.py

```diff
@@ -268,12 +305,27 @@ def _aggregate_features(self, all_features, aggregation_info):
                 agg_alias = agg_info.get("alias", None)
 
                 aggregation_functions = {
-                    sqlglot.exp.Avg: lambda: sum(float(f.get(agg_prop, 0)) for f in features) / len(features),
-                    sqlglot.exp.Sum: lambda: sum(float(f.get(agg_prop, 0)) for f in features),
+                    sqlglot.exp.Avg: lambda: sum(
+                        float(f.get(agg_prop, 0)) for f in features
```

Comment:
Do we still need this casting, if we already casted before? Applies also for the subsequent casts